homework-2/js/main.js

```diff
@@ -0,0 +1,92 @@
+////////////menu///////////
+const iconMenu = document.querySelector('.hamburger-menu');
+  if(iconMenu) {
+    const menuBody = document.querySelector('.menu-body');
+    iconMenu.addEventListener('click', function(e){
+      document.body.classList.toggle('lock');
+      iconMenu.classList.toggle('active');
+      menuBody.classList.toggle('active');
+    });
+  }
+
+//////////sticky header////////////
+window.addEventListener('scroll', function() {
+    var header = document.querySelector('header');
+    header.classList.toggle('sticky-header', window.scrollY > 0);
+});
+
+$(document).ready(function(){ 
+  $('.slider_inner').slick({
+    dots: true,
+    dotsClass: 'my-dots',
+    infinite: false,
+    speed: 300,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    arrows: false,
+    autoplay: true,
+    mobileFirst: true,
+  });
+  $('.slider-logo-single',).slick({
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    arrows: false,
+    fade: true,
+    asNavFor: '.slider-logo-list'
+  });
+
+  $('.slider-logo-list').slick({
+    slidesToShow: 3,
+    slidesToScroll: 1,
+    asNavFor: '.slider-logo-single',
+    dots: true,
+    arrows: false,
+    speed: 300,
+    autoplay: true,
+    focusOnSelect: true,
+    mobileFirst: true,
+    responsive: [
+       {
+        breakpoint: 1200,
```

Comment:
You are using mobile-first. So it is better to start from lowest breakpoint - code will be more clear to understand...

Reply:
done

Reply:
done